TileTowns/Assets/Scripts/GameState/GameState.cs

```diff
@@ -65,6 +69,8 @@ public class GameState : IGameState
     [Inject]
     public GameState(ITurnManager turnManager)
     {
+        Debug.Log("Instantiating game state");
```

Comment:
I'm not sure if you want to keep that log. Just making sure it's not just a forgotten artifact xD

Reply:
Ye.... we can remove it i guess